rental.rb

```diff
@@ -0,0 +1,13 @@
+class Rental
+  attr_accessor :date
```

Comment:
- [ ] good job adding the `initialize` method for `Rental` 👍 however in order to access the `@book` and `@person` outside this method you need to add the attribute accessors for them have a look at [this article](https://github.com/microverseinc/curriculum-ruby/blob/main/oop/articles/oop_principles_examples.md#useful-shortcuts-attr_reader-attr_writer-and-attr_accessor) for more details.

Reply:
Noted and implemented